buildpack/v05/encoderdecoder.go

```diff
@@ -0,0 +1,42 @@
+package v05
+
+import (
+	"os"
+
+	"github.com/BurntSushi/toml"
+
+	"github.com/buildpacks/lifecycle/api"
+	"github.com/buildpacks/lifecycle/buildpack/layertypes"
+)
+
+type EncoderDecoder05 struct {
```

Comment:
Can this struct be private? `encoderDecoder05`

Reply:
The same as https://github.com/buildpacks/lifecycle/pull/537#discussion_r587935792. Am I missing something?

Reply:
Replied in the other thread.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/537/commits/2373122d4321c49d25e9d495e98b6755013b4095